org.opent2t.sample.binaryswitch.superpopular/org.opent2t.sample.binaryswitch.superpopular.json

```diff
@@ -2,21 +2,28 @@
   "id": "http://schemas.microsoft.com/org.opent2t.sample.binaryswitch.superpopular#",
   "$schema": "http://json-schema.org/draft-04/schema#",
   "description": "",
-  "title": "OpenT2T Switch",
+  "title": "OpenT2T SuperPopular Binary Switch",
   "definitions": {
     "org.opent2t.sample.binaryswitch.superpopular": {
       "type": "object",
       "properties": {
         "power": {
-          "$ref": "oic.r.switch.binary.json#/definitions/oic.r.switch.binary"
+          "$ref": "oic.r.switch.binary.json#/definitions/oic.r.switch.binary",
+          "description": "Status of the switch."
         }
       }
     }
   },
   "type": "object",
   "allOf": [
-    { "$ref": "oic.core.json#/definitions/oic.core" },
-    { "$ref": "oic.baseResource.json#/definitions/oic.r.baseresource" },
-    { "$ref": "#/definitions/org.opent2t.sample.binaryswitch.superpopular" }
+    {
+      "$ref": "oic.core.json#/definitions/oic.core"
+    },
+    {
+      "$ref": "oic.baseResource.json#/definitions/oic.r.baseresource"
+    },
+    {
+      "$ref": "#/definitions/org.opent2t.sample.binaryswitch.superpopular"
+    }
```

Comment:
Thanks... this is just the standard auto-format json that vscode does... and since that's how all the other json files in the repo are (supposed to be) formatted I propose we stick with that. I will update the others to be consistent, though, as you point out.